x-pack/test/detection_engine_api_integration/security_and_spaces/tests/import_rules.ts

```diff
@@ -95,8 +97,39 @@ export default ({ getService }: FtrProviderContext): void => {
   const supertest = getService('supertest');
   const log = getService('log');
   const esArchiver = getService('esArchiver');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
 
   describe('import_rules', () => {
+    describe('importing rules with different roles', () => {
+      beforeEach(async () => {
+        await createSignalsIndex(supertest, log);
+      });
+
+      afterEach(async () => {
+        await deleteSignalsIndex(supertest, log);
+        await deleteAllAlerts(supertest, log);
+      });
+      it('should import rules without actions privileges', async () => {
```

Comment:
Could we update this a bit to add context:
```suggestion
      it('should successfully import rules without actions when user has no actions privileges', async () => {
```